### PR TITLE
Move staging area files to project subdirectories 

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#61

### CODE REVIEW

File: scripts/project-sub-dir/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+1. Use the `all_files_by_project.json` as input to `scripts/project-sub-dir/move_files_to_project_dir.py`. 
+
+   ```
+   $ python move_files_to_project_dir.py all_files_by_project.json
+   ```
+   This will copy all the files to project subdirectories to a `prod2` directory in the staging area   
+   
+   This will also output `output.json` file, check for any errors. If there are any errors, fix it then just rerun if not successful.
+
+1. Verify files in `prod2` directory
```
Comment (ESapenaVentura) [inline scripts/project-sub-dir/README.md:58]: I think "how to check" directions should be pointed out here.

One way of checking is repeating steps 2 and 3, if the script works as before it should output an exact copy of the input json map if no entity is there)

File: scripts/project-sub-dir/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Example paths for ingest areas:
+```
+(dev) gs://broad-dsp-monster-hca-dev-ebi-staging/dev/<project-uuid>
+(staging) gs://broad-dsp-monster-hca-dev-ebi-staging/staging/<project-uuid>
+(prod) gs://broad-dsp-monster-hca-prod-ebi-storage/prod/<project-uuid>
+
+```
+
+GitHub Issue: [ebi-ait/dcp-ingest-central#61](https://github.com/ebi-ait/dcp-ingest-central/issues/61)
```
Comment (ESapenaVentura) [inline scripts/project-sub-dir/README.md:20]: Is the link to the GH issue needed in the readme?

File: scripts/project-sub-dir/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+
+## Goal
+Group the existing files in the staging area to be in their own project directory
+
```
Comment (ESapenaVentura) [inline scripts/project-sub-dir/README.md:25]: Kind of missing a "pre-requisites" here, even if it just points out to the SOP on how to set up the gsutil configuration

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]

File: scripts/project-sub-dir/README.md (hunk 1 above)
Comment (aaclan-ebi) [reply scripts/project-sub-dir/README.md:58]: very good point! i'll add that!

File: scripts/project-sub-dir/README.md (hunk 3 above)
Comment (aaclan-ebi) [reply scripts/project-sub-dir/README.md:25]: will add that! thanks!

File: scripts/project-sub-dir/README.md (hunk 2 above)
Comment (aaclan-ebi) [reply scripts/project-sub-dir/README.md:20]: Actually maybe i should just be adding all this info in the ticket? 🤔  not sure where to best document the adhoc steps we do for fixing things. Maybe only the scripts is needed here and how to run those.

Comment (aaclan-ebi) [reply scripts/project-sub-dir/README.md:20]: I think i'll just move this readme in the ticket then.

Comment (aaclan-ebi) [reply scripts/project-sub-dir/README.md:20]: I'll retain the readme here, i think it's fine. And yes i think it's fine to reference the ticket in the README.

**Review by ESapenaVentura — APPROVED** [review]
LGTM 👍

**Pull request merged**